common/messages/heartbeat.go

```diff
@@ -326,7 +327,19 @@ func (m *Heartbeat) FollowerExecute(is interfaces.IState) {
 		if auditServer.GetChainID().IsSameAs(m.IdentityChainID) {
 			if m.IdentityChainID.IsSameAs(is.GetIdentityChainID()) {
 				if m.SecretNumber != is.GetSalt(m.Timestamp) {
-					panic("We have seen a heartbeat using our Identity that isn't ours")
+					lLeaderHeight := is.GetLLeaderHeight()
+					ackChangeValue := is.GetAckChangeValue()
+					ackChangeScheduled := ackChangeValue > is.GetDBHeightAtBoot() && lLeaderHeight <= ackChangeValue
```

Comment:
Loosen the panic condition so the node with the highest chkChange value will not panic right away but waits until this happens again after the block when the brainswap is done.